docs/content/integrations/dbt/reference.mdx

```diff
@@ -532,6 +532,39 @@ def my_dbt_assets(context: AssetExecutionContext, dbt: DbtCliResource):
 
 ---
 
+## Emit dbt table schema as materialization metadata <Experimental />
```

Comment:
Nitpick: I think calling this "column schema" or "column metadata" would be more clear. "Table schema" could be interpreted as "the database schema that the table lives inside".